### PR TITLE
#6 Fetch Twitter dataset

### PR DESCRIPTION
**Pull request by piatra**
Fetched most popular words.
Adds method to scrape alexa given an url.

### CODE REVIEW

**Comment by piatra** [comment]
@tiriplicamihai please review.

File: src/data_set_builder/data_set_builder.py [hunk 1]
```diff
@@ -5,13 +5,33 @@
 import requests
 from sqlalchemy.orm import sessionmaker
 
+# scraping modules
+import sys
+sys.path.append("../demographic_scraper/")
```
Comment (tiriplicamihai) [inline src/data_set_builder/data_set_builder.py]: Didn't PYTHONPATH var from Dockerfile solve this?

File: src/data_set_builder/data_set_builder.py [hunk 2]
```diff
@@ -5,13 +5,33 @@
...
 
 from models import create_db_tables, db_connect, WebsitesContent
 from google import search
 
 MAX_RESULTS_LIMIT = 50
 
 
+def crawl_for_demographics(url):
+    """Fetch demographics from (currently just) Alexa.com"""
+    configure_logging({'LOG_FORMAT': '%(levelname)s: %(message)s'})
```
Comment (tiriplicamihai) [inline src/data_set_builder/data_set_builder.py]: This should be done in `main`

File: src/twitter_data_set_builder/most_popular_words [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+apple
+rip
+dah
+gracias
+art
+kita
+went
+elsorteodev
+fight
+nigga
```
Comment (tiriplicamihai) [inline src/twitter_data_set_builder/most_popular_words:14]: lol

**Comment by tiriplicamihai** [comment]
lgtm

File: src/data_set_builder/data_set_builder.py (hunk 1 above)
Comment (piatra) [inline src/data_set_builder/data_set_builder.py]: You're right. I updated the code now. It wasn't entirely clear to me how python module system worked out paths and the fact that we have a child and parent folder hierarchy with the same name made things more confusing.

**Pull request merged**